samcli/commands/build/command.py

```diff
@@ -205,6 +205,14 @@ def cli(
     """
     # All logic must be implemented in the ``do_cli`` method. This helps with easy unit testing
 
+    track_metric(
```

Comment:
can we move telemetry related code to its own decorator?

Reply:
yeah, I'll revert this part, and do another PR to extend `track_command` to track options